.github/workflows/cd-publish.yml

```diff
@@ -3,38 +3,98 @@ name: CD - Publish for Blazor / FluentUI
 on:
   workflow_dispatch
 
+env:
+  OUTPUT_PATH: ${{ github.workspace }}/.nupkg
+  NUGET_SOURCE: "https://api.nuget.org/v3/index.json"
+  DOTNET_VERSION: "5.0.5"
+  GITHUB_SOURCE: "https://nuget.pkg.github.com/Azure/index.json"
+  CONFIGURATION: Release
+
 jobs:
-  publish:
+  build job:
     name: setup, build, validate, and publish
     runs-on: windows-latest
     if: contains('nicholasrice,awentzel,eisenbergeffect,Ibrahimmaga', github.actor)
 
     steps:
     - uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-        
-    - name: Setup .NET
+    - name: Setup .NET Core ${{ env.DOTNET_VERSION }}
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 5.0.x
-        
-    - name: Restore dependencies
+        dotnet-version: ${{env.DOTNET_VERSION}}
+    - name: Install dependencies
       run: dotnet restore
-      
     - name: Build
-      run: dotnet build --no-restore
-      
-    - name: Test
-      run: dotnet test --no-build --verbosity normal
-      
-    - name: Publish NuGet
-      uses: brandedoutcast/publish-nuget@v2.5.5
+      run: dotnet build --configuration ${{ env.CONFIGURATION }} --no-restore
+    - name: Package
+      run: dotnet pack --configuration ${{ env.CONFIGURATION }} --no-build --output ${{ env.OUTPUT_PATH }} -p:PackageVersion="${GITHUB_REF#refs/*/v}"
+        working-directory: ${{ github.workspace }}/src/Microsoft.Fast.Components.FluentUI/Microsoft.Fast.Components.FluentUI.csproj
+    - name: Upload
+      uses: actions/upload-artifact@v2
+      with:
+        name: packages
+        path: ${{ env.OUTPUT_PATH }}
+
+  publish-nuget:
+    steps:
+      shell: powershell
+    - task: DotNetCoreCLI@2
+      displayName: '.NET Pack'
+      inputs:
+      command: custom
+      custom: 'pack'
+      projects: src/Microsoft.Fast.Components.FluentUI/Microsoft.Fast.Components.FluentUI.csproj
+      arguments: '-c ${{ parameters.configuration }} -o $(Build.ArtifactStagingDirectory) --no-build --include-symbols /p:PackageVersion=$(Version)$(VersionSuffix) -v Quiet'
```

Comment:
why should we include symbols?

Reply:
we don't have to, we are not using the symbol